CODEOWNERS

```diff
@@ -9,4 +9,14 @@ packages/webview_flutter/**                    @bparrishMines
 
 # Sub-package-level rules. These should stay last, since the last matching
 # entry takes precedence.
-packages/**/*_web/**                           @ditman
+packages/**/*_web/**                                     @ditman
```

Comment:
Let's add platform headings since this could get pretty messy as soon as another platform did the same thing. So

```
# Sub-package-level rules. These should stay last, since the last matching
# entry takes precedence.

# - Web
packages/**/*_web/**                                     @ditman

# - Android
...
```

Reply:
done